lib/process-services-cloud/src/lib/form/components/widgets/radio-buttons/radio-buttons-cloud.widget.spec.ts

```diff
@@ -74,6 +75,23 @@ describe('RadioButtonsCloudWidgetComponent', () => {
         expect(field.updateForm).toHaveBeenCalled();
     });
 
+    it('should call rest api only when url is present and field type is rest', () => {
+        const getFieldConfig = (optionType: string, restUrl: string) => new FormFieldModel(new FormModel(), { optionType, restUrl });
+        spyOn(formCloudService, 'getRestWidgetData').and.returnValue(of([]));
+
+        widget.field = getFieldConfig('manual', 'fake-rest-url');
+        widget.ngOnInit();
+        expect(formCloudService.getRestWidgetData).not.toHaveBeenCalled();
+
+        widget.field = getFieldConfig('rest', null);
+        widget.ngOnInit();
+        expect(formCloudService.getRestWidgetData).not.toHaveBeenCalled();
+
+        widget.field = getFieldConfig('rest', 'fake-rest-url');
+        widget.ngOnInit();
+        expect(formCloudService.getRestWidgetData).toHaveBeenCalled();
```

Comment:
maybe we can move it to separate function to avoid duplications? same in the dropdown

Reply:
It's easier to understand when we don't extract it to the function, especially toHaveBeenCalled explain directly what we want to achieve